stdlib/3/builtins.pyi

```diff
@@ -472,7 +472,7 @@ class slice:
     start = 0
     step = 0
     stop = 0
-    def __init__(self, start: int, stop: int = 0, step: int = 0) -> None: ...
+    def __init__(self, start: Optional[int], stop: Optional[int] = 0, step: Optional[int] = 0) -> None: ...
```

Comment:
The default really should be `...`, in stubs the value of the default is irrelevant, only its presence is used.